xml/deployment_scenarios.xml

```diff
@@ -375,6 +396,15 @@
     automatically or can be forced via the CLI. You can then copy this data to
     your trusted storage at any point and update the internal mirror.
    </para>
+   <note>
+    <title>Required Modules</title>
+    <para>
+     &rmtool; provides software in so called modules. You must enable all the
+     default modules for &sle; and in addition the <literal>Containers</literal>
+     module. This module needs to be mirrored inside the airgapped network
+     in order to provide all necessary software for other parts of this scenario.
```

Comment:
We don't need the containers module to be mirrored inside of the RMT server of the airgapped network.

Neither `skopeo` nor `helm-mirror` are required inside of the airgappend network

Reply:
Ok will remove this from the instructions

Reply:
the registry package is coming from the containers module.
so we do need it mirrored inside the airgap network otherwise you can't install the registry on the internal mirror.

https://www.suse.com/documentation/sles-15/book_sles_docker/data/preparation.html
https://www.suse.com/documentation/sles-15/book_sles_docker/data/sec_docker_registry_installation.html

I will remove skopeo and helm-mirror requirement from the internal mirror.